editor/editor_properties.cpp

```diff
@@ -1856,13 +1857,15 @@ EditorPropertyTransform::EditorPropertyTransform() {
 ////////////// COLOR PICKER //////////////////////
 
 void EditorPropertyColor::_color_changed(const Color &p_color) {
-
-	emit_changed(get_edited_property(), p_color, "", true);
+	if (!Input::get_singleton()->is_mouse_button_pressed(BUTTON_LEFT))
+		emit_changed(get_edited_property(), p_color, "", true);
+	else
+		get_edited_object()->set(get_edited_property(), p_color);
 }
 
 void EditorPropertyColor::_popup_closed() {
 
-	emit_changed(get_edited_property(), picker->get_pick_color(), "", false);
+	// emit_changed(get_edited_property(), picker->get_pick_color(), "", false);
```

Comment:
I guess you might just remove the whole function (mind the reference in the `connect()` call)